src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs

```diff
@@ -34,10 +34,11 @@ private static void Initialize()
 
             // this will initialize ApplicationManager and Environment if they haven't yet
             var logPath = Environment.LogPath;
+            var logInfo = "";
```

Comment:
If you're only using this variable inside the block, you shouldn't declare outside the block